runtime/doc/news.txt

```diff
@@ -234,6 +234,9 @@ UI
   which controls the tool used to open the given path or URL. If you want to
   globally set this, you can override vim.ui.open using the same approach
   described at |vim.paste()|.
+• The |ui-popupmenu| now supports cascading highlight styles. |hl-PmenuSel|
```

Comment:
This is wrong. `ui-popupmenu` disables Nvim's rendering of the popupmenu.

Reply:
Oh oops! Thanks. Just `|popup-menu|` then?